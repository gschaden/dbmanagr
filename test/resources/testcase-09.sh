db-navigator.py -s skiline@localhost/
