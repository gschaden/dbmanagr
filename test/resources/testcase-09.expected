Id	Title	Subtitle	Autocomplete	
skiline@localhost/hikeline	skiline@localhost/hikeline/	Database	skiline@localhost/hikeline/
skiline@localhost/postgres	skiline@localhost/postgres/	Database	skiline@localhost/postgres/
skiline@localhost/skiline	skiline@localhost/skiline/	Database	skiline@localhost/skiline/
skiline@localhost/skiline_test	skiline@localhost/skiline_test/	Database	skiline@localhost/skiline_test/

